perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/GroupsManagerBlImpl.java

```diff
@@ -1687,14 +1684,28 @@ protected boolean hasCandidateExistingMember(Candidate candidate, RichMember ric
 				categorizeMembersForSynchronization(sess, actualGroupMembers, candidates, candidatesToAdd, membersToUpdate, membersToRemove);
 			}
 
+			//Remove presented members in group who are not presented in synchronized ExtSource
```

Comment:
// Remove members from group who are not present in synchronized ExtSource

Reply:
Done.